modules/ob-installing-builds-using-CLI.adoc

```diff
@@ -0,0 +1,37 @@
+// This module is included in the following assembly:
+//
+// builds/installing-openshift-builds.adoc
+
+:_content-type: PROCEDURE
+[id="installing-builds-using-CLI_{context}"]
+= Installing {builds-shortname} by using the CLI
+
+You can also install {builds-title} by using the CLI.
+
+.Procedure
+
+. Create a `sub.yaml` subscription object file to subscribe a namespace to the {builds-title} Operator, as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1 
+kind: Subscription 
+metadata: 
+  name: openshift-builds-operator 
+  namespace: openshift-operators 
+spec: 
+  channel: <channel-name> 
+  name: openshift-builds-operator
+  source: redhat-operators 
```

Comment:
@jkhelil can you please confirm what will be the `source` once the operator is GAed?
For the current operator (that we have deployed for testing), the source is `openshift-builds-operator`, so, will it remain the same or will be `redhat-operators`?

Reply:
redhat-operators, we should add a legend here because this catlogsource doesnt necessary exist
please have a look to what is done for pipelines
https://docs.openshift.com/container-platform/4.9/cicd/pipelines/installing-pipelines.html